package.json

```diff
@@ -5,11 +5,12 @@
   "main": "index.js",
   "scripts": {
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "test": "set NODE_ENV=test&& nyc mocha --require @babel/register 'src/**/**.test.js' --no-timeout ",
-    "dev-start": "nodemon --exec babel-node src/index.js",
+    "pretest": "NODE_ENV=test yarn drop && yarn migrate",
+    "test": "NODE_ENV=test nyc mocha --recursive --exit --require @babel/register --require @babel/polyfill 'src/**/**.test.js' --no-timeout",
+    "dev-start": "NODE_ENV=development nodemon --exec babel-node src/index.js",
     "build": "babel src --out-dir build",
     "start": "node build/index.js",
-    "drop": "sequelize-cli db:migrate:undo:all",
+    "drop": "./node_modules/.bin/babel-node ./node_modules/.bin/sequelize db:migrate:undo:all",
```

Comment:
Name this script `undo-migration` (it is more descriptive) instead of `drop`

Reply:
I've resolved it. Thanks.